spec/grape/validations_spec.rb

```diff
@@ -574,7 +574,7 @@ def validate_param!(attr_name, params)
         # NOTE: with body parameters in json or XML or similar this
         # should actually fail with: children[parents][name] is missing.
         expect(last_response.status).to eq(400)
-        expect(last_response.body).to eq('children[1][parents] is missing')
+        expect(last_response.body).to eq('children[1][parents] is missing, children[0][parents][1][name] is missing, children[0][parents][1][name] is empty')
```

Comment:
To be honest, it is something I would like to avoid. If a parent node is missing, why do we need to check children? :thinking:

Reply:
I would put it in "nice to have". Generally, you could see it both ways. If the info is not there, you see an error, then you add a parent, then it tells you that you're still missing children. Get frustrated, repeat.